api/internal/handler/cache_verify/cache_verify_test.go

```diff
@@ -0,0 +1,104 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cache_verify
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/apisix/manager-api/internal/core/storage"
+	"github.com/apisix/manager-api/internal/core/store"
+	"github.com/apisix/manager-api/internal/log"
+	"github.com/shiningrush/droplet"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestHandler_CacheVerify(t *testing.T) {
+	andyStr := `{"username":"andy","plugins":{"key-auth":{"key":"key-of-john"}},"create_time":1627739045,"update_time":1627744978}`
+	var andyObj interface{}
+	err := json.Unmarshal([]byte(andyStr), &andyObj)
+	if err != nil {
+		log.Errorf("unmarshal error :: %s", err)
+		return
+	}
+	brokenAndyStr := `{"username":"andy","plugins":{"key-auth":{"key":"key-of-john"}},"create_time":1627739046,"update_time":1627744978}`
+	var brokenAndyObj interface{}
+	err = json.Unmarshal([]byte(brokenAndyStr), &brokenAndyObj)
+	if err != nil {
+		log.Errorf("unmarshal error :: %s", err)
+		return
+	}
+	consumerPrefix := "/apisix/consumers/"
+
+	tests := []struct {
+		caseDesc                 string
+		listInput                string
+		listRet                  []storage.Keypair
+		getInput                 string
+		getRet                   interface{}
+		wantInconsistentConsumer int
+	}{
+		{
+			caseDesc:  "consistent",
+			listInput: consumerPrefix,
+			listRet: []storage.Keypair{
+				{
+					Key:   consumerPrefix + "andy",
+					Value: andyStr,
+				},
+			},
+			getInput:                 "andy",
+			getRet:                   andyObj,
+			wantInconsistentConsumer: 0,
+		},
+		{
+			caseDesc:  "inconsistent",
+			listInput: consumerPrefix,
+			listRet: []storage.Keypair{
+				{
+					Key:   consumerPrefix + "andy",
+					Value: andyStr,
+				},
+			},
+			getInput:                 "andy",
+			getRet:                   brokenAndyObj,
+			wantInconsistentConsumer: 1,
+		},
+	}
+
+	for _, tc := range tests {
+		t.Run(tc.caseDesc, func(t *testing.T) {
+			mockConsumerCache := store.MockInterface{}
+			mockEtcdStorage := storage.MockInterface{}
+			mockEtcdStorage.On("List", context.TODO(), consumerPrefix).Return(tc.listRet, nil)
+			// for any other type of configs,etcd.List just return empty slice,so it will not check further
+			mockEtcdStorage.On("List", context.TODO(), mock.Anything).Return([]storage.Keypair{}, nil)
+
+			mockConsumerCache.On("Get", tc.getInput).Return(tc.getRet, nil)
+			handler := Handler{consumerStore: &mockConsumerCache, etcdStorage: &mockEtcdStorage}
+			rs, err := handler.CacheVerify(droplet.NewContext())
+			assert.Nil(t, err, nil)
+			v, ok := rs.(OutputResult)
+			assert.True(t, ok, true)
+			assert.Equal(t, v.Items.Consumers.InconsistentCount, tc.wantInconsistentConsumer)
+
+		})
+	}
+
+}
```

Comment:
What is the test coverage?

Reply:
![image](https://user-images.githubusercontent.com/17563754/131543517-be69be31-8ca0-4ce7-9bb0-2019c5939429.png)